x-pack/plugins/maps/public/classes/layers/vector_layer/vector_layer.tsx

```diff
@@ -76,6 +76,10 @@ interface JoinState {
   propertiesMap?: PropertiesMap;
 }
 
+interface EditableData {
```

Comment:
This type is only used in a isEditable method. Lets just inline it there to make things easier to follow and avoid cluttering interface definitions.